### PR TITLE
Add Edge versions for BaseAudioContext API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for Microsoft Edge for the `BaseAudioContext` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.3.3).  Results are manually confirmed for accuracy.

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/BaseAudioContext

### CODE REVIEW

**Comment by foolip** [comment]
I believe that `BaseAudioContext` was introduced later than `AudioContext` itself and that some members were moved. Since this is what tells you what's supported on `AudioContext` for lots of methods I think we need to change the tests for this to use an `AudioContext` instance. I don't trust results from the prototype tests that we now have.

**Comment by foolip** [comment]
I've sent https://github.com/foolip/mdn-bcd-collector/pull/971 to test this using an `AudioContext` instance. @vinyldarkscratch can you check the results against https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/BaseAudioContext after that's merged?

**Comment by vinyldarkscratch** [comment]
Thanks @foolip!  I've updated this PR with collector v1.3.3 results.

File: api/BaseAudioContext.json
Comment (foolip) [inline api/BaseAudioContext.json:14]: https://mdn-bcd-collector.appspot.com/tests/api/BaseAudioContext returns false for Edge 18, but I understand you had to set this to "12" since some subfeatures were in Edge 12.

I think we should instead have a `partial_implementation` from Edge 12-18 with a note saying that while the BaseAudioContext interface itself wasn't supported, most of it members were on AudioContext. Since BaseAudioContext is an abstract interface it doesn't matter.

@ddbeck @Elchi3 for suggestions on note wording. We could use the same wording for other similar situations in https://github.com/mdn/browser-compat-data/issues/7517.

File: api/BaseAudioContext.json
Comment (foolip) [inline api/BaseAudioContext.json:513]: https://developer.mozilla.org/en-US/docs/Web/API/ConstantSourceNode#browser_compatibility could also be updated to "79" then.

File: api/BaseAudioContext.json
Comment (Elchi3) [reply api/BaseAudioContext.json:14]: Since I started work on the mixin business I wonder about how much documentation should sometimes hide weird facts. Here, I think, it might be another of those cases, where adding this fact to compat data or to docs is just confusing people and adds to the perception of the API surface as a highly fragile construct and the web being always incompatible due to many differences across engines.

Which is a long way of saying: Is this really that interesting and needs to be talked about? If the functionality is there and it is working, why do you think support is partial only?

Comment (Elchi3) [reply api/BaseAudioContext.json:14]: Ah, I now see this is about `BaseAudioContext` itself, and not its members. Still, it seems `partial_support` potentially alarms people about this being incompatible when it isn't really?

Comment (foolip) [reply api/BaseAudioContext.json:14]: Indeed this is only about what to do with the `api.BaseAudioContext` entry. Although abstract interfaces like `BaseAudioContext` or `SVGGeometryElement` do appear on the prototype chain and can be observed, I wouldn't say that their existence is especially interesting.

My concern is leaving some kind of trace of the fact that `BaseAudioContext` wasn't originally there, so that if in the future any tooling tests for its existence and suggests our data is wrong, we won't forget. So it's really notes for maintainers I want here.

That being said, I'd imagine two support statements, where the most recent has no notes at all, so this wouldn't end up looking especially scary in the resulting MDN tables.

Comment (Elchi3) [reply api/BaseAudioContext.json:14]: > That being said, I'd imagine two support statements, where the most recent has no notes at all, so this wouldn't end up looking especially scary in the resulting MDN tables.

This sounds reasonable to me. Can you do that here?

> So it's really notes for maintainers I want here.

`maintainer_notes` as a new BCD property then? I guess there is an issue for that? (/me dreams of JSON with comments, ah well).

Comment (ddbeck) [reply api/BaseAudioContext.json:14]: I couldn't find an open one, so I opened https://github.com/mdn/browser-compat-data/issues/9223

Comment (foolip) [reply api/BaseAudioContext.json:14]: For this PR, should we use a note in the interim?

Comment (ddbeck) [reply api/BaseAudioContext.json:14]: Yeah, I think a note for now and revisiting this when we do something about #9223 is the way to go.

Comment (foolip) [reply api/BaseAudioContext.json:14]: OK, so I suggest a range for Edge 12-18 with the note "The <code>BaseAudioContext</code> interface itself is not present, but many of the methods are available on the <code>AudioContext</code> interface." Then a range starting at Edge 79 with no notes.

File: api/BaseAudioContext.json
Comment (foolip) [reply api/BaseAudioContext.json:513]: I've sent https://github.com/mdn/browser-compat-data/pull/9278 to make it so.

File: api/BaseAudioContext.json
Comment (foolip) [inline api/BaseAudioContext.json:20]: @ddbeck @Elchi3 WDYT?

Comment (foolip) [reply api/BaseAudioContext.json:20]: I guess I we should use `partial_implementation` too?

Comment (Elchi3) [reply api/BaseAudioContext.json:20]: I don't think anyone would look for it, but I guess given `BaseAudioContext` is not observable in these versions, let's add `partial_implementation`.

Comment (foolip) [reply api/BaseAudioContext.json]: Alright, done!

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**
